src/csharp/Grpc.Core/RpcException.cs

```diff
@@ -56,5 +71,18 @@ public Status Status
                 return status;
             }
         }
+
+        /// <summary>
+        /// Gets the call trailing metadata.
+        /// Trailers only have meaningful content for client-side calls (in which case they represent the trailing metadata sent by the server when closing the call).
+        /// Instances of <c>RpcException</c> thrown by the server-side part of the stack will have trailers always set to empty.
+        /// </summary>
+        public Metadata Trailers
```

Comment:
I'd just use `public Metadata Trailers { get; }` and assign directly to this in the constructor - and to the same with `Status` - unless you particularly need to be pre-C#-6 friendly.